components/phttp/ammo/simple/raw/provider.go

```diff
@@ -0,0 +1,98 @@
+package raw
+
+import (
+	"bufio"
+	"context"
+	"io"
+	"log"
+	"os"
+
+	"github.com/facebookgo/stackerr"
+	"github.com/spf13/afero"
+
+	"github.com/yandex/pandora/components/phttp/ammo/simple"
+)
+
```

Comment:
It would be great have short ammo format specification here, for better understanding what happens)